bellows/zigbee/zcl/__init__.py

```diff
@@ -238,7 +247,7 @@ def _update_attribute(self, attrid, value):
         self.listener_event('attribute_updated', attrid, value)
 
     def log(self, lvl, msg, *args):
-        msg = '[0x%04x:%s:0x%04x] ' + msg
+        msg = '[0x%04x:%s:0x%04x] ' + str(msg)
```

Comment:
I prefer not to do this. I'll put the comments for it where you log the exception.